### ISSUE TITLE
Make Navigation Bar Transparent?

### ISSUE THREAD

**Issue by spenceropope** [initialxy/cordova-plugin-themeablebrowser#20]
Is there any way to set the navigation bar to transparent so that there will not be a margin at the top of the page?

**Comment by initialxy** [comment]
Are you trying to get rid of toolbar altogether, so the WebView basically takes the whole height? If that's the case, then you can just set toolbar height to 0, and don't specify any other properties. eg.

    cordova.ThemeableBrowser.open('http://apache.org', '_blank', {
        toolbar: {
            height: 0
        }
    });

**Comment by spenceropope** [comment]
I want the buttons to still show up, just directly over the content.

**Comment by initialxy** [comment]
Unfortunately this is not a currently supported feature. I will mark this issue as an enhancement and evaluate its feasibility at a later time.

**Comment by spenceropope** [comment]
Do you have any idea how I could edit the Objective C code to achieve this? I'm thinking that altering the properties for `self.toolbar` may be a feasible route.

**Comment by initialxy** [comment]
If you are looking for a way to hack together iOS, then what you want to do is to move the UIWebView up. In fact if you look at the existing code, you can see that its moving UIWebView down as well as reducing its height by toolbar's height. Ctrl + f `webViewBounds` in [CDVThemeableBrowser.m](https://github.com/initialxy/cordova-plugin-themeablebrowser/blob/master/src/ios/CDVThemeableBrowser.m) and it should be pretty self explanatory.

eg. Comment out line [589](https://github.com/initialxy/cordova-plugin-themeablebrowser/blob/master/src/ios/CDVThemeableBrowser.m#L589), [971](https://github.com/initialxy/cordova-plugin-themeablebrowser/blob/master/src/ios/CDVThemeableBrowser.m#L971), [972](https://github.com/initialxy/cordova-plugin-themeablebrowser/blob/master/src/ios/CDVThemeableBrowser.m#L972)

Why are there so many different instance? Because the code base was inherited from [InAppBrowser](https://github.com/apache/cordova-plugin-inappbrowser) and some legacy codes were deliberately kept so that merging from InAppBrowser would be easier.

Then you just need to set toolbar's background colour to transparent. Notice that colour value always supports RGBA. eg.

    cordova.ThemeableBrowser.open('http://apache.org', '_blank', {
        toolbar: {
            color: '#00000000'
        },
        ...
    });

**Comment by initialxy** [comment]
Added a new `fullscreen` flag in release 0.2.7. Please see [README.md](https://github.com/initialxy/cordova-plugin-themeablebrowser/blob/master/README.md).

**Comment by meirgottlieb** [comment]
Hello, 

After this change the statusbar color does not seem to be setting properly. I'm setting the toolbar and statusbar to the same color but they do not appear as the same color on the device. I'm using iOS. I noticed this after updating to the latest version today.

Thanks

**Comment by initialxy** [comment]
Could you please attach a screenshot? Feel free to mask any confidential info.

**Comment by meirgottlieb** [comment]
Here is a screenshot showing the toolbar and statusbar. 

![screenshot 2015-06-11 15 25 55](https://cloud.githubusercontent.com/assets/2698576/8118766/4ffe8960-104e-11e5-8242-eeba4a36613a.png)

And here is the configuration I'm using:

```
        var loginWindow = cordova.ThemeableBrowser.open(loginUrl + '?returnTo=' + encodeURIComponent(returnTo), '_blank', {
            statusbar: {
                color: '#EEEEEEFF'
            },
            toolbar: {
                height: 44,
                color: '#EEEEEEFF'
            },
            title: {
                color: '#333333FF',
                staticText: "Log in"
            },
            closeButton: {
                image: 'close_button',
                imagePressed: 'close_button',
                align: 'left',
                event: 'closePressed'
            }
        });
```

**Comment by meirgottlieb** [comment]
FYI, I just rolled back to version 0.2.6 and here is how it looks in the older version:

![screenshot 2015-06-11 15 30 43](https://cloud.githubusercontent.com/assets/2698576/8118881/e6452f1e-104e-11e5-918c-33a7636097ed.png)

**Comment by initialxy** [comment]
Thanks, I will investigate.

**Comment by initialxy** [comment]
Fixed in version 0.2.9. Please note that due to [Cordova's migration to npm](http://cordova.apache.org/announcements/2015/04/21/plugins-release-and-move-to-npm.html), plugin ID has been updated to `cordova-plugin-themeablebrowser`.
